### ISSUE TITLE
Should tC render the Greek and Hebrew word lists delineated by the \wg and \wh markers?

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#6567]
![image](https://user-images.githubusercontent.com/8171759/69098435-33a57700-0a1e-11ea-8df2-e6e6aed4d6b2.png)
[usfm body testE.usfm.zip](https://github.com/unfoldingWord/translationCore/files/3860958/usfm.body.testE.usfm.zip)

**Comment by jag3773** [comment]
Yes, I believe so.  I think those are related to the https://ubsicap.github.io/usfm/characters/index.html#w-w marker.  The idea is to mark certain words in your text that should appear in a glossary as well.

@RobH123 is that how you understand it?

**Comment by RobH123** [comment]
I'm not completely aware of the context here but I think you're correct. I don't find the USFM3 spec totally clear here, but certainly words surrounded by /w ... /w* are considered part of the actual Biblical text. I assume that the contents of \wg and \wh marker sets are not.

But no! https://github.com/ubsicap/usfm/blob/master/sty/usfm.sty#L2590 shows all those styles as PUBLISHABLE VERNACULAR, in other words they are considered part of the canonical text. Unless the stylesheet is wrong (unlikely), I'm guessing that you're looking for a marker that's marked UNPUBLISHABLE.
